boardgamegeek/api.py

```diff
@@ -888,17 +888,23 @@ def game(self, name=None, game_id=None, choose=BGGChoose.FIRST, versions=False,
         while added_items and len(game.comments) < total:
             page += 1
 
+            params['page'] = page
             xml_root = request_and_parse_xml(self.requests_session,
                                              self._thing_api_url,
-                                             params={"id": game_id,
-                                                     "pagesize": 100,
-                                                     "comments": 1,
-                                                     "page": page})
+                                             params=params,
```

Comment:
I am reusing the params structure built above, but with a change just to `page`. This was needed to have `rating_comments` set in the params structure. I could have just added that to the new params structure that is defined inline here, but it seemed less error-prone to reuse the original params structure. However, I'm not sure if it might cause problems to include all of the elements in the original params structure here (including the ones that aren't actually necessary in this call)?

Reply:
This call to BGG is only meant to get comments, without videos, history, etc. (whatever else the user requested). That's why I created a new structure; if you reuse `params`, the api will return extra data you don't need (and for each requested page, I guess).